.envrc

```
@@ -0,0 +1 @@
+use flake
```

It might be worth not assuming everybody is using flakes:
```suggestion
if nix flake info &>/dev/null; then
  use flake
else
  use nix
fi
```

---

+1